generic/proof-shell.el

```diff
@@ -477,6 +477,11 @@ process command."
 	(erase-buffer)
 	(proof-shell-set-text-representation)
 
+        ;; If one quits Proof General, it's clear that the proof
+        ;; assistant needs to be killed - don't ask.
```

Comment:
On the one hand I agree with your remark;
but on the other hand I believe changing would be not "uniform" w.r.t. other emacs features?

e.g. if we do `M-x shell RET`
then `C-x C-s`, exactly as you say, "if we quit emacs, it's clear that the nested shell needs to be killed", but emacs will still ask for confirmation.

Would you agree to customize this?